lib/patterns.coffee

```diff
@@ -83,10 +83,9 @@ exports.login = (callback) ->
 #		console.log(token)
 ###
 exports.loginWithToken = (callback) ->
-	form.ask
-		label: 'What\'s your token? (visible in the preferences page)'
-		name: 'token'
-		type: 'text'
+	form.askAuthToken
```

Comment:
Since using a new prompt just for this function is an edge case, and someone without knowing the reasons behind it wouldn't be able to figure out why we're doing it, there should be a comment explaining what is happening here (and potentially linking to the GitHub issue in Resin CLI that reports the repeating issue).